lightning/src/util/config.rs

```diff
@@ -126,6 +126,30 @@ pub struct ChannelHandshakeConfig {
 	///
 	/// [`KeysInterface::get_shutdown_scriptpubkey`]: crate::chain::keysinterface::KeysInterface::get_shutdown_scriptpubkey
 	pub commit_upfront_shutdown_pubkey: bool,
+
+	/// The Proportion of the channel value to configure as counterparty's channel reserve,
+	/// i.e., `their_channel_reserve_satoshis` for both outbound and inbound channels.
+	///
+	/// `their_channel_reserve_satoshis` is the minimum balance that the other node has to maintain
+	/// on their side, at all times.
+	/// This ensures that if our counterparty broadcasts a revoked state, we can punish them by
+	/// claiming at least this value on chain.
+	///
+	/// Channel reserve values greater than 30% could be considered highly unreasonable, since that
+	/// amount can never be used for payments.
+	/// Also, if our selected channel reserve for counterparty and counterparty's selected
+	/// channel reserve for us sum up to equal or greater than channel value, channel negotiations
+	/// will fail.
+	///
+	/// Note: Versions of LDK earlier than v0.0.104 will fail to read channels with any channel reserve
+	/// other than the default value.
+	///
+	/// Default value: 1% of channel value, i.e., configured as 10,000 millionths.
+	/// Minimum value: If the calculated proportional value is less than 1000 sats, it will be treated
+	///    			   as 1000 sats instead, which is a safe implementation-specific lower bound.
```

Comment:
I'm a bit confused, but I'm not sure when the mix between tabs and spaces after the /// got added again? I thought that they were addressed previously. Text alignment should be done with spaces only (and not tabs which is confusing as the rest of the code base uses tabs for indentation).

Reply:
had fixed it, idk how it got added again, maybe due to my habit.